css/build.css

```diff
@@ -120,187 +121,203 @@
 }
 /* METRO SIZES */
 .panels.size11 {
-  width: 145px;
+  width: 50%;
   height: 145px;
 }
 
 .panels.size21 {
-  width: 300px;
+  width: 100%;
   height: 145px;
 }
 
 .panels.size22 {
-  width: 300px;
+  width: 100%;
   height: 300px;
 }
 @media (min-width: 420px) {
   .panels.size11 {
-    width: 160px;
+    width: 29%;
```

Comment:
I'm seeing you use %'s like 29% and 30%. Why is it not 33.3%? You've used 50% for 2-column layouts so it seems that 33.3% or 25% etc. should be used.

Reply:
@tonytlwu Yes, you are right we should use those sizes,  that was my mistake. But to understand:
Mobile:
Small - two in a row
Medium - one in a row
Large - one in a row and two in a column 

Tablet/Desktop
Small - three in a row
Medium - two in a row
Large - two in a row and 2 in a column

Are those correct sizes?

Reply:
Yes they sound about right. See if this changes how the layouts behave and confirm with new videos of all sizes and viewports.

Reply:
@tonytlwu While I was testing the new solution I've noticed that when we have 4 items in resolution more than 1300px we hot such result
![image](https://user-images.githubusercontent.com/53430352/96886057-e4689180-148b-11eb-87f9-78ee0bdcfc98.png)

Is it desirable behavior or its a bug and we need to fix it?

Reply:
I think the screenshot you've shown here, given all the items are meant to the same size, then they should look the same, i.e. it looks like a bug to me. With all the latest changes including [these suggestions](https://github.com/Fliplet/fliplet-widget-metro/pull/28#discussion_r505711541), hopefully it'll resolve the problem?

Reply:
@tonytlwu Yes, we can resolve it. I'll do the fix now and create a new PR.